<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAircraftCombinedTypeSelector</class>
 <widget class="QFrame" name="CAircraftCombinedTypeSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="hl_AircraftCombinedType" stretch="1,3,1,3">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="le_CombinedType">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Enter combined type, e.g. L2J, S1P</string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="placeholderText">
      <string>L1P</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cb_Type">
     <property name="toolTip">
      <string>Type</string>
     </property>
     <item>
      <property name="text">
       <string>Helicopter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Glider</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Seaplane</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Landplane</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tilt wing</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Unspecified</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cb_EngineCount">
     <property name="toolTip">
      <string>Engines count</string>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cb_EngineType">
     <property name="toolTip">
      <string>Engine</string>
     </property>
     <item>
      <property name="text">
       <string>Electric</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Jet</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Piston</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Turboprop</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Unspecified</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
